my.css

```diff
@@ -0,0 +1,107 @@
+.main {
+	position: relative;
+	max-width: 1200px;
+	height: 100%;
+	
+}
+.burgermenu {
+	position: absolute;
+	right: 0;
+	top: 0;
+	width: 70px;
+	height: 70px;
+	background-color: #e3e1e1;
+
+}
+.burger {
+	position: absolute;
+	top: 20px;
+    left: 18px;
+	width: 40px;
+    height: 4px;
+    margin-top: 5px;
+    background: #000;	
+}
+.burger::before {
+	content: '';
+	position: absolute;
+	width: 40px;
+	top: 13px;
+	left: 0;
+	height: 5px;
+	background: #000;
+}
+.promo {
+    width: 1440px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)